### PR TITLE
netstat: add route print example

### PR DESCRIPTION
**Pull request by cam8001**
I like printing my route table, so updated tldr to show how I do it :)

- [*] The page has 8 or fewer examples.

- [*] The PR is appropriately titled:  
      `<command name>: add page` for new pages, or `<command name>: <description of changes>` for pages being edited.

- [*] The page follows the [contributing](https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md) guidelines.

### CODE REVIEW

**Comment by agnivade** [comment]
Did you intentionally make the change only on the osx page ? Because we have a linux counterpart too, and since `-nr` exists, on linux too, I would like both pages to be consistent.

**Comment by cam8001** [comment]
Linux added! Thanks for confirming @sbrl.

File: pages/linux/netstat.md [hunk 1]
```diff
@@ -14,18 +14,14 @@
 
 `netstat -t`
 
-- Display PID and program names:
+- Display PID and program names for a specific port:
 
-`netstat -p`
+`netstat -p {PROTOCOL}`
```
Comment (agnivade) [inline pages/linux/netstat.md]: -p is for program name. The current description is correct.

File: pages/linux/netstat.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # netstat
 
-> Displays network-related information such as open connections, open socket ports etc.
+> Displays network-related information such as open connections, open socket
```
Comment (sbrl) [inline pages/linux/netstat.md]: I'm not sure there's meant to be a line break here.

File: pages/osx/netstat.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # netstat
 
-> Displays network-related information such as open connections, open socket ports etc.
+> Displays network-related information such as open connections, open socket
```
Comment (sbrl) [inline pages/osx/netstat.md]: ....or here

File: pages/linux/netstat.md [hunk 4]
```diff
@@ -29,3 +29,7 @@
 - List listening TCP and UDP ports (+ user and process if you're root):
 
 `netstat -lepunt`
+
+- Print the route table:
```
Comment (sbrl) [inline pages/linux/netstat.md]: Perhaps this will read better?

```suggestion
- Print the routing table:
```

File: pages/osx/netstat.md [hunk 5]
```diff
@@ -21,3 +21,7 @@
 - List information continuously:
 
 `netstat -c`
+
+- Print the route table:
```
Comment (sbrl) [inline pages/osx/netstat.md]:
```suggestion
- Print the routing table:
```

Comment (cam8001) [reply pages/osx/netstat.md]: I've personally never heard it referred to as a 'routing table', but Google (and Wikipedia) suggest that that term is more common. Updated.

**Pull request merged**